src/containers/org/components/dataset/DataSetActionButton.js

```diff
@@ -126,13 +144,18 @@ const DataSetActionButton = ({ dataSet, isAtlas, organizationId } :Props) => {
             horizontal: 'right',
             vertical: 'top',
           }}>
-        <MenuItem disabled={!isOwner || !isAtlas || isPromoted} onClick={handleOpenPromote}>
+        <MenuItem disabled={!isOrgOwner || !isAtlas || isPromoted} onClick={handleOpenPromote}>
           {
             isPromoted
               ? 'Promoted'
               : 'Promote Data Set'
           }
         </MenuItem>
+        <AssembleMenuItem
+            disabled={!hasMaterialize || isAtlas}
+            entitySetId={dataSetId}
+            isAssembled={isAssembled}
+            organizationId={organizationId} />
```

Comment:
I thought we decided it should not be visible if it's an atlas data set, i.e. only show for entity sets?